### ISSUE TITLE
Create standard WDL file for testing purposes

### ISSUE THREAD

**Issue by bansals10** [uchicago-cs/chiventure#317]
Currently, we are using variations of the sample WDL file to test the loading and validation functions. I am creating a standardized WDL file that contains the proper syntax and definitions according to the WDL specification.

**Comment by bansals10** [comment]
I created a new YAML file that follows the WDL format specified in wdl.md within the docs directory. In essence, this is a game that can be parsed by and loaded into game-state structures using the libobj and libwdl functions. Because different modules were previously using separate files for testing, this file will serve as the central file upon which all testing will occur. I am creating a PR to include this updated file, and the FILE_PATH macro will be modified to reference this file in the wdl_common.h file.

**Comment by bansals10** [comment]
Preemptively closed this issue. Need to complete the text_fail and text_success fields within the sample.yaml file.

**Comment by bansals10** [comment]
Closing statement from above: 
"I created a new YAML file that follows the WDL format specified in wdl.md within the docs directory. In essence, this is a game that can be parsed by and loaded into game-state structures using the libobj and libwdl functions. Because different modules were previously using separate files for testing, this file will serve as the central file upon which all testing will occur. I am creating a PR to include this updated file, and the FILE_PATH macro will be modified to reference this file in the wdl_common.h file."

Added the following fields:
- attributes
- associated attribute values
- conditions (id, attribute, value)
- effects (id, attribute, value)
- text_fail
- text_success
